docs/relying-parties/reference/metrics-for-relying-parties.md

```diff
@@ -4,7 +4,29 @@ title: Metrics for Relying Parties
 sidebar_label: Metrics
 ---
 
-Last updated: `January 26th, 2022`
+Last updated: `March 15th, 2023`
+
+## Subscriptions Services Data Sources
+Subscription Platform provides product metrics by obtaining subscriptions data from the following sources: Stripe, mobile app store providers, and FxA logs.
+
+### Stripe
+The bulk of Subscriptions Services data comes from Stripe and is synced to BigQuery using FiveTran and its Stripe connector. This data reflects the current subscription state of all active subscriptions paid for via Stripe or PayPal. It includes information, such as the customer's start and end dates of a subscription, etc.
```

Comment:
Help: what other information is included?

Reply:
The following paragraph lists some other customer information that's included.

In addition to subscriptions and customers we also have products, plans, invoices, cards, charges, discounts, coupons, and promotion codes.